client/src/Components/Card/FuelLeftCard.js

```diff
@@ -3,8 +3,8 @@ import { Card, Icon } from 'semantic-ui-react';
 import FuelEconomyCard from './FuelEconomyCard';
 
 class FuelLeftCard extends Component {
-  render(props) {
-    if (!this.props.lastRefillOdo[0], !this.props.refillData[0]) {
+  render() {
+    if ((!this.props.lastRefillOdo[0], !this.props.kmsLeft)) {
```

Comment:
did you mean to use the comma operator? I think you might've intended to use `||`